forui/CHANGELOG.md

```diff
@@ -25,10 +38,15 @@
 * **Breaking** Rename `FSelectMenuTile.groupController` to `FSelectMenuTile.selectController`.
 * **Breaking** Rename `FSelectMenuTile.menuTileBuilder` to `FSelectMenuTile.menuBuilder`.
 
+* **Breaking** Rename `FTappable` to `FTappable.static`.
+* **Breaking** Rename `FTappable.animated` to `FTappable`.
```

Comment:
nitpick: would be nice to leave a note on how existing users should migrate.

Reply:
The only thing they need to do is rename it though. What do you have in mind?